usr/share/rear/prep/GNU/Linux/305_include_s390_tools.sh

```diff
@@ -0,0 +1,29 @@
+#
+#  s390 zIPL boot loader and grubby for configuring boot loader`
+
+test -d $VAR_DIR/recovery || mkdir -p $VAR_DIR/recovery
+
+local bootdir="$( echo -n /boot/ )"
+test -d "$bootdir" || $bootdir='/boot/'
+
+# cf. https://github.com/rear/rear/issues/2137
+# findmnt is used the same as grub-probe to find the device where /boot is mounted
+# example
+# findmnt -no SOURCE --target /boot
+# --> /dev/dasda1
+#
+# sles will need to be looked at:
+# findmnt returns --> /dev/dasda3[/@/.snapshots/1/snapshot]
+# it's possible that sles can use 300_include_grub_tools.sh instead of this file
+if has_binary findmnt ; then
+    echo 'run findmnt'
```

Comment:
Regarding the `echo` command see my
https://github.com/rear/rear/pull/2142/files#r282782104

Reply:
removed to be consistent with 300_include_grub_tools.sh
does it make sense to rename 305_include_s390_tools.sh to 300_include_s390_tools.sh since they do the same thing but for diff ARCH?

Reply:
For now leave it as it i.e. with different numbers.
When you use same numbers it makes no difference.
With same numbers also both get sourced and executed.